### ISSUE TITLE
Narrow down issues with callstack tests

### ISSUE THREAD

**Issue by vringar** [mozilla/OpenWPM#557]
Currently we know the following:
- Running plain FF makes the stackDump API behave as expected.
- Running `demo.py` (or tests)  and the JS Instrument enabled makes the stackDump API behave as expected.
- Running  `demo.py` and no JS leads to the actors not being created in normal windows and only the children being created in privileged windows (DevTool, about:debugging) but even there no callstacks get captured

Further suggested exploration:
- Set up a minimal selenium project and test if the extension works in that context (aka debugging configwise but with selenium prefs)
- Run with a reduced config for non-debugging (prod) setup
- Inject more complex contentscript when running with OpenWPM

Resolution options:
- Find solution on friday
- Programmatically enforce in `deployFirefox.py` that the callstacks code doesn't run without both HTTP instrument and the JS instrument

**Comment by englehardt** [comment]
@vringar is this still an issue?

**Comment by vringar** [comment]
Yes, the error still happens on [this version](https://github.com/mozilla/OpenWPM/tree/1d01187cef8194578033602be7d565f8bee234e7) aka our current version.

**Comment by englehardt** [comment]
We still aren't sure why this is required. For now, we're going to force both instruments to be enabled when the callstack instrument is enabled. We can do this when we validate the browser params config (see #495).
